pyproject.toml

```diff
@@ -42,6 +44,7 @@ path = "src/hipercow/__about__.py"
 dependencies = [
   "pytest",
   "pytest-cov",
+  "responses",
```

Comment:
Should this be in the other batch of dependencies above (line 29ish?) - CI tests all fine, but I got "No module named 'responses'" running hatch test, so had a guess and moved it up, which sorted things for me.